modules/cookbook/pages/recipe-clm.adoc

```diff
@@ -0,0 +1,81 @@
+[[cookbook-clm]]
+= Content Lifecycle Management
+
+If you are managing a lot of clients and you need to apply customized packages to them, you can use CLM to manage your packages.
+Content lifecycle management (CLM) allows you to customize and test packages before updating production clients.
+It is also useful if you need to apply updates during a limited maintenance window.
+
+
+
+== Problem
+
+Content lifecycle management allows you to select software channels as sources, adjust them as required for your environment, and thoroughly test them before installing onto your production clients.
```

Comment:
And now the question is whether to use the abbrev always or not always ;)

Reply:
Spell it out the first time in each section, than abbrev after that?

Reply:
sounds like a plan :)